lib/matplotlib/table.py

```diff
@@ -211,12 +243,17 @@ def __init__(self, ax, loc=None, bbox=None, **kwargs):
         self.set_clip_on(False)
 
         self._cachedRenderer = None
+        self._cellType = 'default'
```

Comment:
underscores instead of camel case please.  Yes, the code base is mixed, but lets not make it worse with new additions.